### ISSUE TITLE
Feature Request: Pre-loading Banner and MREC Ads

### ISSUE THREAD

**Issue by yo1106** [AppLovin/AppLovin-MAX-Flutter#232]
### Feature Description

**Description:**
It would be highly beneficial to have a feature that allows pre-loading of banner ads in the AppLovin MAX Flutter SDK. Currently, pre-loading support exists for interstitial ads but is missing for banner ads. On slower devices, loading video banner ads can take substantial time, leading to a poor user experience.

**Use Case:**
In scenarios where we anticipate needing a banner ad (e.g., upon loading a specific view or during navigation), pre-loading the banner ad can improve performance by reducing the time taken to display the ad.

**Expected Behavior:**
1. Ability to invoke a method for pre-loading a banner ad.
2. The application should be able to check if the banner ad is pre-loaded and ready to display.

**Example API Proposal:**
```dart
MaxAdView preloadedBannerAd = MaxAds.preloadBannerAd(adUnitId: 'YOUR_BANNER_AD_UNIT_ID');
if (preloadedBannerAd.isReady) {
    preloadedBannerAd.show();
}
```

**Benefit:**
Enhances the user experience by reducing visible loading times for banner ads, especially on lower-performance devices.
Ensures that ads are ready to be shown immediately when required.

**Additional Note:**
A similar feature request has been raised for the React Native plugin: [AppLovin-MAX-React-Native/issues/57](https://github.com/AppLovin/AppLovin-MAX-React-Native/issues/57).

**Comment by alhiwatan** [comment]
@yo1106, thank you for filing an RFE for preloading banner ads. We will consider adding it to our queue.

**Comment by alhiwatan** [comment]
@yo1106 and users,

We have released applovin_max 3.11.1, which now supports preloading banners and MRECs. Please refer to our documentation [here](https://developers.applovin.com/en/max/flutter/ad-formats/banner-mrec-ads#ad-preloading).

As this is the initial release of this feature, we would greatly appreciate any feedback!
